src/style_corect.css

```diff
@@ -0,0 +1,86 @@
+/* @font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+} */
+
+.search-bar__input {
+  /* display: block;
+  width: 100%; */
+  margin-top: 20px;
+}
+
+.search-bar__input--big {
+  /* height: 70px; */
+
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: 25px center;
+}
```

Comment:
The top indent of 20px for the big search bar is missing. Please ensure that there is a top indent of 20px for the big search bar as specified in the task requirements.